bin/celeste_mcmc.jl

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env julia
```

Comment:
That's my fault, I just wanted to get Andy start quickly. But yeah, I think an `MCMC` module is the way to go.